packages/syntax-core/src/RadioButton/RadioButton.tsx

```diff
@@ -12,18 +13,19 @@ const RadioButton = ({
   disabled = false,
   error = false,
   label,
+  name,
   onChange,
   size = "md",
   value = "",
```

Comment:
I wanted to make the default value the same value as `label` but I'm not sure of that's possible in typescript?